en/Plugins/Search.md

```diff
@@ -1,82 +1,98 @@
-- `Collapse results` will toggle between just showing matching note names and showing the lines in which matches appear. These extended results can be toggled for each note by clicking on the folding triangle next to the file name.
-- `Show more context` will expand the display of the matches to show more text around the match.
-- `Change sort order` sorts the results by various orders, similar to how files are sorted in the [[File explorer]].
+- `meeting -work` returns files that contain `meeting` but not `work`.
 
-## Copy search results
+> [!tip] Explain search term
+> If you need to troubleshoot a complex search term, you can click **Explain search term** in the Search pane for an explanation of your search term.
 
-To quickly and easily take your search results to a list, use the "copy search result" option.
+## Search operators
 
-You can customize whether you want to show paths, which link style to use (wikilink or Markdown link), and how you want your list to appear.
+Search operators enable more fine-grained search queries to filter your results even more.
 
-## Embed search results
+Some operators even allow you to add a nested search term within parentheses, for example: `task:(call OR email)`.
 
-You can embed search results in a note.
+| Search operator | Description                                                                                                                                                                                                          |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `file:`         | Find text in filename.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |
+| `path:`         | Find text in file path.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |
+| `content:`      | Find text in file content.<p/>Example: `content:"happy cat"`.                                                                                                                                                        |
+| `match-case:`   | Case-sensitive match.<p/>Example: `match-case:HappyCat`.                                                                                                                                                             |
+| `ignore-case:`  | Case-insensitive match.<p/>Example: `ignore:case:ikea`.                                                                                                                                                              |
```

Comment:
"ignore:case" => "ignore-case" in example

Reply:
Good eye! 👁️